### ISSUE TITLE
Question about eval

### ISSUE THREAD

**Issue by ChrisLLLLL** [mkhorr/supergluetraining#4]
Hello, If I want to evaluate a new pair of pictures.How will i get the input.txt? such as "path_image_A path_image_B exif_rotationA exif_rotationB [KA_0 ... KA_8] [KB_0 ... KB_8] [T_AB_0 ... T_AB_15]".
I dont know how get a 3x3 intrinsics matrix of image0: K0, a 3x3 intrinsics matrix of image1: K1 , and a 4x4 matrix of the relative pose extrinsics T_0to1.
Looking forward to your reply!

**Comment by mkhorr** [comment]
Hello,

Look under "Evaluation mode" in the original SuperGluePretrainedNetwork README as this repo uses the same evaluation code. Also look at assets/scannet_sample_pairs_with_gt.txt for an example of the input file format.

The intrinsic matrices and relative pose matrices are groundtruth information that comes with the scannet dataset. You can either construct the file yourself using whatever image pairs you want to test with, or you can convert the binary training files to the txt format using the pairfiles_bin_to_txt function in utils.py. There's also a train_val_split function in utils.py that extracts a test set by taking 100 pairs from each scene, but it doesn't remove the entries from the the training set, so you may need to modify it to suit your needs.
